packages/driver/src/multi-domain/communicator.ts

```diff
@@ -105,9 +105,21 @@ export class PrimaryDomainCommunicator extends EventEmitter {
    * @param {string} event - the name of the event to be sent.
    * @param {any} data - any meta data to be sent with the event.
    */
-  toSpecBridge (event: string, data?: any) {
+  toAllSpecBridges (event: string, data?: any) {
```

Comment:
Opinions on function naming? this is kinda awkward, but also clear

Reply:
Works for me.

Reply:
Could go with some sort of `broadcast` naming.

Reply:
I was thinking some sort of broadcast, but wasn't sure if that was more clear or just a more enjoyable word to say.